linera-core/src/unit_tests/client_tests.rs

```diff
@@ -1643,76 +1680,1188 @@ where
+        .contains(&Operation::System(SystemOperation::ReadBlob {
+            blob_id: blob0_id
+        })));
+    assert!(validator_manager
+        .requested_proposed
+        .unwrap()
+        .content
+        .block
+        .operations
+        .contains(&Operation::System(SystemOperation::PublishBlob {
+            blob_id: blob1_id
+        })));
+    assert!(validator_manager.requested_locked.is_none());
+
+    let validator_manager = builder
+        .node(2)
+        .chain_info_with_manager_values(chain_id2)
+        .await
+        .unwrap()
+        .manager;
+    assert!(validator_manager
+        .requested_proposed
+        .clone()
+        .unwrap()
+        .content
+        .block
+        .operations
+        .contains(&Operation::System(SystemOperation::ReadBlob {
+            blob_id: blob0_id
+        })));
+    assert!(validator_manager
+        .requested_proposed
+        .unwrap()
+        .content
+        .block
+        .operations
+        .contains(&Operation::System(SystemOperation::PublishBlob {
+            blob_id: blob1_id
+        })));
+    assert!(validator_manager.requested_locked.is_none());
+
+    builder.set_fault_type([2], FaultType::Offline).await;
+    builder.set_fault_type([0, 1, 3], FaultType::Honest).await;
+
+    let bt_certificate = client2
+        .burn(None, Amount::from_tokens(1), UserData::default())
+        .await
+        .unwrap()
+        .unwrap();
+
+    let hashed_certificate_values = client2
+        .read_hashed_certificate_values_downward(bt_certificate.value.hash(), 2)
+        .await
+        .unwrap();
+
+    // Latest block should be the burn
+    assert!(hashed_certificate_values[0]
+        .inner()
+        .block()
+        .unwrap()
+        .operations
+        .contains(&Operation::System(SystemOperation::Transfer {
+            owner: None,
+            recipient: Recipient::Burn,
+            amount: Amount::from_tokens(1),
+            user_data: UserData::default(),
+        })));
+
+    // Previous should be the `ChangeOwnership` operation, as the blob operations shouldn't be executed here.
+    assert!(hashed_certificate_values[1]
+        .inner()
+        .block()
+        .unwrap()
+        .operations
+        .contains(&owner_change_op));
+    Ok(())
+}
+
+#[test_case(MemoryStorageBuilder::default(); "memory")]
+#[cfg_attr(feature = "storage-service", test_case(ServiceStorageBuilder::new().await; "storage_service"))]
+#[cfg_attr(feature = "rocksdb", test_case(RocksDbStorageBuilder::new().await; "rocks_db"))]
+#[cfg_attr(feature = "dynamodb", test_case(DynamoDbStorageBuilder::default(); "dynamo_db"))]
+#[cfg_attr(feature = "scylladb", test_case(ScyllaDbStorageBuilder::default(); "scylla_db"))]
+#[test_log::test(tokio::test)]
+async fn test_blob3<B>(storage_builder: B) -> anyhow::Result<()>
+where
+    B: StorageBuilder,
+    ViewError: From<<B::Storage as Storage>::StoreError>,
+{
+    // Configure a chain with two regular and no super owners.
```

Comment:
That comment should probably be further down.

Reply:
This was a left over when I first copy pasted another test to write the first one of these 😅 will just remove